### ISSUE TITLE
easy internal linking to available content

### ISSUE THREAD

**Issue by creative-resort** [netrivet/prophoto-issues#741]
Please make automatic linking available to

- posts
- pages
- galleries etc (available permalinks)

in all places where currently URLs can only be manually be entered, e.g. 

- graphics widget
- text widget editor > link functionality

**Comment by brianium** [comment]
@creative-resort - do you have some examples of how you would like this to work?

**Comment by creative-resort** [comment]
yes, very much like in the grid settings and in the menu item settings
so a post, page, gallery etc can be selected
